content/sensu-core/1.4/guides/monitor-the-monitor.md

```diff
@@ -0,0 +1,197 @@
+
+#### Monitoring Sensu Server Locally
+
+Monitoring the host that the `sensu-server` process runs on should be done just like any other node in your infrastructure. This includes, but not limited to, checks and metrics for [CPU][1], [memory][2], [disk][3], and [networking][4]. You can find more plugins at the [Sensu Community Homepage][5].
+
+#### Monitoring Sensu Server Remotely
+
+To monitor the `sensu-server` Server process, you will need to do so from an independent Sensu stack. This can be done by reaching out to Sensu's [API health endpoint][6] and using the [check-http sensu plugin][7] with the following check definition.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_sensu_server_port": {
+      "command": "check-http.rb -h remote-api-hostname -P 4567 -p /health?consumers=1 --response-code 204",
+      "subscribers": [
+        "monitor_remote_sensu_api"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+### Monitoring Sensu API{#monitoring-sensu-api}
+
+To monitor the `sensu-api` API service, you will need to do so from an independent Sensu stack. This can be done by reach out to the port that the API is listening on using the [check-port sensu plugin][8] with the following check definition.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_sensu_api_port": {
+      "command": "check-ports.rb -H remote-sensu-server-hostname -p 4567",
+      "subscribers": [
+        "monitor_remote_sensu_api"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+### Monitoring Uchiwa{#monitoring-uchiwa-dashboard}
+
+*Method 1: Monitor Uchiwa with a process check*
+
+To monitor the Uchiwa Dashboard, you will need to check for two processes named `uchiwa` using [check-process sensu plugin][9] using the following check definition. This check will return a check result with status 2 if less than 2 processes are running with the string `/opt/uchiwa/bin/uchiwa`. We look for two as there is a parent and child process for the `uchiwa service`
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_uchiwa_process": {
+      "command": "/opt/sensu/embedded/bin/check-process.rb -p /opt/uchiwa/bin/uchiwa -C 2",
+      "subscribers": [
+        "monitor_local_uchiwa_processes"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+*Method 2: Monitor Uchiwa with a remote network port check*
+
+You can also monitor the `uchiwa` Dashboard with a remote port check using the [check-port sensu plugin][8] with the following check definition.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_uchiwa_port": {
+      "command": "check-ports.rb -H uchiwa-remote-hostname -p 3000",
+      "subscribers": [
+        "monitor_remote_uchiwa_port"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+_PRO TIP: Use both checks for complete monitoring of your Uchiwa dashboard: catch when both processes are not running and when a firewall port is not open_
```

Comment:
This reads a bit strangely. Maybe "Use both checks for complete monitoring of Uchiwa. This way, you are able to catch when the processes are not running and when a firewall port is not open"